src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/IncompleteDirectives_Runtime.codegen.cs

```diff
@@ -81,9 +81,7 @@ internal sealed class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_I
         #nullable restore
 public global::Microsoft.AspNetCore.Mvc.ViewFeatures.ViewDataDictionary<TestFiles_IntegrationTests_CodeGenerationIntegrationTest_IncompleteDirectives> ViewData => (global::Microsoft.AspNetCore.Mvc.ViewFeatures.ViewDataDictionary<TestFiles_IntegrationTests_CodeGenerationIntegrationTest_IncompleteDirectives>)PageContext?.ViewData!;
 #nullable disable
-        #nullable restore
-public TestFiles_IntegrationTests_CodeGenerationIntegrationTest_IncompleteDirectives Model => ViewData.Model!;
-#nullable disable
+        public TestFiles_IntegrationTests_CodeGenerationIntegrationTest_IncompleteDirectives Model => ViewData.Model;
```

Comment:
This looks suspicious. Previously the `Model` property here was nullable-enabled, now it's nullable-disabled, isn't that a regression?

Reply:
The Model property in this case isn't coming from the user, if you look it's the type of name of the file which is generated code, declared outside of the users nullability annotations. So, it's correct that it shouldn't be nullable annotated.